app/services/mentions/create_all.rb

```diff
@@ -53,7 +53,7 @@ def authored_by?(user, notifiable)
 
     def delete_mentions_removed_from_notifiable_text(users)
       mentions = @notifiable.mentions.where.not(user_id: users).destroy_all
-      Notification.remove_all(notifiable_ids: mentions.pluck(:id), notifiable_type: "Mention") if mentions.present?
+      Notification.remove_all(notifiable_ids: mentions.pluck(:id), notifiable_type: "Mention") if mentions.present? # rubocop:disable Rails/PluckId
```

Comment:
Absolutely not a problem/blocker but to avoid the disable message in this particular case maybe `mentions.map(&:id)` could be used?

Reply:
let me check if that works

Reply:
it worked, thanks! 

see https://github.com/forem/forem/pull/9537/commits/f4b5adf18f49f60636b8446e5e2631e7f1cf2497 and https://github.com/forem/forem/pull/9537/commits/22ec65c5af8db8ab966b141631ba8df6b54fe395